web/cypress/e2e/route/create-route-with-search-service-and-set-priority.cy.js

```diff
@@ -171,7 +172,7 @@ context('Create Route with search service name', () => {
     cy.contains(data.serviceName).siblings().contains('Delete').click();
     cy.contains('button', 'Confirm').click();
     cy.get(selector.notification).should('contain', data.deleteServiceSuccess);
-
+    cy.wait(4000);
```

Comment:
You can click here to remove the tip in the upper right corner, we should try not to waste time by waiting as much as possible:

```
  cy.get('.ant-notification-close-icon').click().should('not.exist');
```